rke/k8s_version_info.go

```diff
@@ -203,6 +234,18 @@ func loadK8sVersionInfo() map[string]v3.K8sVersionInfo {
 			MinRancherVersion: "2.4.6-rc0",
 			MinRKEVersion:     "1.1.4-rc0",
 		},
+		// The Calico/Canal template in this version use functions that are only available in RKE v1.0.0 and up
+		// This version includes nodelocal dns only available in RKE v1.0.7 and up
+		"v1.18.6-rancher1-1": {
+			MinRancherVersion: "2.4.6-rc0",
```

Comment:
This should be the same as the one in 2.4 branch, I believe it's 2.4.4-rc0, same with the rke min version